src/attr/exceptions.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import, division, print_function
+
+
+class FrozenInstanceError(AttributeError):
```

Comment:
Raise this at https://github.com/hynek/attrs/pull/60/files#diff-21756223332b04173cec8910fda6655bR576 instead of AttributeError?